servicetest/timingprofiling/test_profiling.py

```diff
@@ -93,7 +93,7 @@ def run_test(num_starts=3):
         for app in range(0, num_starts):
             """ Start numStarts apps in softwarecontainer.
             """
-            print "Start app " + str(app)
+            print("Start app " + str(app))
```

Comment:
Use .format instead of +
https://softwareengineering.stackexchange.com/questions/304445/why-is-s-better-than-for-concatenation